src/services/destination/postTransformation.ts

```diff
@@ -152,6 +161,12 @@ export default class DestinationPostTransformationService {
         authErrorCategory: errObj.authErrorCategory,
       }),
     } as DeliveryResponse;
+
+    // for transformer-proxy to maintain contract
+    const { response } = error;
+    if (response) {
```

Comment:
`DeliveryResponse` + `response` == ErrorResponse 
Do we need two different types ?

Reply:
cc @aashishmalik

Reply:
reverted the typeConversion to `ErrorResponse`, will do this after more tests